config/graph.mainnet.yml

```diff
@@ -131,8 +133,8 @@ contracts:
     init:
       controller: "${{Controller.address}}"
     calls:
-      - fn: "setIssuanceRate"
-        issuanceRate: "1000000011247641700" # per block increase of total supply, blocks in a year = 365*60*60*24/13
+      - fn: "setIssuancePerBlock"
+        issuancePerBlock: "114155251141552511415" # per block increase of total supply, blocks in a year = 365*60*60*24/12
```

Comment:
And this is missing from all three arbitrum config files (assuming we are starting with a 5% issuance on L2).

Reply:
Config files reflect 100% issuance on L1 - as the idea for now is that the 5-95 split will be executed after deployment. I think we can update the config files _after_ we've done that on the relevant networks